docs/content/latest/contribute/core-database/run-unit-tests.md

```diff
@@ -86,3 +87,30 @@ org.yb.client.TestYBClient.txt
 {{< note title="Note" >}}
 The YB logs are contained in the output file now.
 {{< /note >}}
+
+### YSQL Java tests
+
+YSQL java tests are in `java/yb-pgsql/src/test/java/org/yb/pgsql/`.  They can be run as:
+```bash
+./yb_build.sh --java-test TestPgTruncate
```

Comment:
Actually, this is a shorthand that could possibly conflict with stuff in `java/yb-cql`, for example.  Use `--java-test org.yb.pgsql.TestPgTruncate`.